src/murmur/Meta.cpp

```diff
@@ -598,15 +593,23 @@ bool MetaParams::loadSSLSettings() {
 		qWarning("MetaParams: TLS cipher preference is \"%s\"", qPrintable(pref.join(QLatin1String(":"))));
 	}
 
-	qscCert         = tmpCert;
-	qlCA            = tmpCA;
-	qlIntermediates = tmpIntermediates;
-	qskKey          = tmpKey;
-	qbaDHParams     = tmpDHParams;
-	qsCiphers       = tmpCiphersStr;
-	qlCiphers       = tmpCiphers;
-
-	qmConfig.insert(QLatin1String("certificate"), QString::fromUtf8(qscCert.toPem()));
+	if (!tmpCert.isNull()) {
```

Comment:
As I see it, `tmpCert` will always be null at this point, because it has only been default constructed but never used in any way (that I can see) before this check. Could this be a bug? :thinking: